app/services/fan_out_on_write_service.rb

```diff
@@ -57,16 +57,20 @@ def render_anonymous_payload(status)
   def deliver_to_hashtags(status)
     Rails.logger.debug "Delivering status #{status.id} to hashtags"
 
+    prefix = ENV.fetch('REDIS_PUBSUB_PREFIX') { '' }
+
     status.tags.pluck(:name).each do |hashtag|
-      Redis.current.publish("timeline:hashtag:#{hashtag}", @payload)
-      Redis.current.publish("timeline:hashtag:#{hashtag}:local", @payload) if status.local?
+      Redis.current.publish("#{prefix}timeline:hashtag:#{hashtag}", @payload)
+      Redis.current.publish("#{prefix}timeline:hashtag:#{hashtag}:local", @payload) if status.local?
```

Comment:
Should we wrap `Redis.current` with a wrapper class if we add prefixes? It seems like there's enough repeated logic here to warrant a separate class specifically for doing Redis operations rather than calling `Redis.current` directly everywhere